code/controllers/subsystems/records.dm

```diff
@@ -181,54 +179,27 @@
 /datum/controller/subsystem/records/proc/reset_manifest()
 	manifest.Cut()
 
-// The one and only method for showing a pop-up crew manifest (browser) window
-/proc/open_crew_manifest(var/mob/user, var/OOC = FALSE)
-	if(!user)
-		return
-	var/const/windowname = "manifest"
-	var/dat = {"<h2 style="text-align: center">Crew Manifest</h2>"}
-	dat += SSrecords.get_manifest(OOC)
-	send_theme_resources(user)
-	user << browse(enable_ui_theme(user, dat), "window=[windowname];size=450x600")
-	return windowname
-
-/datum/controller/subsystem/records/proc/get_manifest(var/OOC = FALSE)
-	var/const/style = {"
-			.manifest {border-collapse: collapse; width: 100%}
-			.manifest td, th {border: 1px solid #DEF; background-color: white; color:black; padding: .25em}
-			.manifest th {height: 2em; background-color: #3F668F; color: white}
-			.manifest tr.head th {background-color: #006E7A;}
-			.manifest td:first-child {text-align: right}
-			.manifest tr.alt td {background-color: #DEF}
-		"}
-	var/dat = {"
-			<head><style>[style]</style></head>
-			<table class="manifest">
-			<tr class='head'><th>Name</th><th>Rank</th><th>Activity</th></tr>
-		"}
-	var/even = 0
-	var/list/isactive = new()
-	for(var/mob/M in player_list)
-		if (OOC)
-			if(M.client && M.client.inactivity <= 10 * 60 * 10)
-				isactive[M.real_name] = "Active"
-			else
-				isactive[M.real_name] = "Inactive"
-		else
-			isactive[M.real_name] = 0
+/datum/controller/subsystem/records/CanUseTopic(var/mob/user, var/datum/topic_state/state = default_state) // this is needed because VueUI closes otherwise
+	if(isnewplayer(user))
+		return STATUS_INTERACTIVE
+	if(isobserver(user))
+		return STATUS_INTERACTIVE
+	if(issilicon(user)) // silicons have the show manifest verb
+		return STATUS_INTERACTIVE
```

Comment:
Should this, then, either call their verb or display a message telling them to use their verb? Not sure.

Reply:
Oh no, this is for when this code is called _by_ their verb. It's there just to remind people that silicons do have this despite also having a computer inside.